packages/web3torrent/src/clients/payment-channel-client.ts

```diff
@@ -83,6 +84,9 @@ export class PaymentChannelClient {
   async initialize() {
     await this.channelClient.provider.mountWalletComponent(process.env.WALLET_URL);
     await this.initializeHubComms();
+    if (process.env.NODE_ENV === 'production') {
+      Sentry.setUser({id: this.channelClient.signingAddress});
+    }
```

Comment:
@snario According to https://docs.sentry.io/enriching-error-data/additional-data/?platform=javascript

> Each of these is optional, but one *must* be present for Sentry to collect data about the user:

So I pass the address as the user `id`